docs/design/features/framework-version-resolution.md

```diff
@@ -147,12 +147,12 @@ Pros
 Cons
 * Testing behavior of new releases with pre-release versions is not fully possible (see below).
 * Some special cases don't work:
-One special case which would not work:  
-*Component A which asks for `2.0.0 LatestMajor` is loaded first on a machine which has `3.0.0` and also `3.1.0-preview` installed. Because it's the first in the process it will resolve the runtime according to the above rules - that is prefer release version - and thus will select `3.0.0`.  
-Later on component B is loaded which asks for `3.1.0-preview LatestMajor` (for example the one in active development). This load will fail since `3.0.0` is not enough to run this component.  
+One special case which would not work:
+*Component A which asks for `2.0.0 LatestMajor` is loaded first on a machine which has `3.0.0` and also `3.1.0-preview` installed. Because it's the first in the process it will resolve the runtime according to the above rules - that is prefer release version - and thus will select `3.0.0`.
```

Comment:
This was in a new line, but after removing the trailing whitespace from the previous line it's now not. Do you want it to be in a separate line?

Reply:
Having it all on one line is fine. I would make the raw text to match the rendered view and put it all on the same line in the raw text as well.

Also, could you please change `Some special cases don't work:` -> `Some special cases don't work.` while you are on it.